web/api/token_middleware.go

```diff
@@ -19,6 +18,8 @@ const (
 	TokenTypeAccess = "access"
 	// TokenTypeRefresh is a refresh token type.
 	TokenTypeRefresh = "refresh"
+	// TokenTypeTFAPreauth is an 2fa preauth token type.
+	TokenTypeTFAPreauth = "2fa-preauth"
```

Comment:
why do we need these duplications of tokens? The same here `jwt/service/token_service.go`

Reply:
agree with @Compaurum 
I would suggest not to introduce new token type, but grant limited scope for this token. Like limit role or, well `scope: [tfa]` array in token payload. So the user with this token could only go to one endpoint

Reply:
@Compaurum this is a historical issue, I will fix it.

Reply:
@SuddenGunter amazing proposal, I will do the scope instead.